src/main/java/seedu/address/logic/parser/CreateCommandParser.java

```diff
@@ -68,8 +70,11 @@ public CreateCommand parse(String args) throws ParseException {
         Details details = argMultimap.getValue(PREFIX_DETAILS).isPresent()
                 ? ParserUtil.parseDetails(argMultimap.getValue(PREFIX_DETAILS).get())
                 : null;
+        Set<Interaction> interaction = new HashSet<Interaction>();
+        // command does not allow adding interactions right away
 
-        Person person = new Person(name, phone, email, address, tagList, telegram, profession, income, details);
+        Person person = new Person(name, phone, email, address, tagList, telegram, profession, income, details,
+                interaction);
```

Comment:
```suggestion
        Person person = new Person(name, phone, email, address, tagList, telegram, profession, income, details,
                interaction);
```

Reply:
I think the constructor should not have the interaction

Reply:
This is useful for when we are copying data onto CSV and loading from CSV files later?

Reply:
Okay, removed!